### PR TITLE
Adding shashankchakelam's SSH key

### PR DESCRIPTION
**Pull request by shashankchakelam**
Adding shashankchakelam's SSH key

### CODE REVIEW

**Comment by mayankkapoor** [comment]
please test this

File: hiera/data/env/at.yaml [hunk 1]
```diff
@@ -7,8 +7,8 @@ rjil::ceph::osd::osd_journal_size: 2
 # Only Qemu is supported on virtual testing environment
 nova::compute::libvirt::libvirt_virt_type: qemu
 
-rjil::system::accounts::active_users: [soren,bodepd,hkumar,jenkins,consul,pandeyop,jaspreet,vivek,ahmad,vaidy,himanshu,rohit,sanjayu,alokjani,amar,ynshenoy,abhishekl,mayankkapoor,soumit,anshup,varunarya,prashant,punituee,saju,ajayaa]
-rjil::system::accounts::sudo_users: [soren,bodepd,hkumar,jenkins,consul,pandeyop,jaspreet,vivek,ahmad,vaidy,himanshu,rohit,sanjayu,alokjani,amar,ynshenoy,abhishekl,mayankkapoor,soumit,anshup,varunarya,prashant,punituee,saju,ajayaa]
+rjil::system::accounts::active_users: [soren,bodepd,hkumar,jenkins,consul,pandeyop,jaspreet,vivek,ahmad,vaidy,himanshu,rohit,sanjayu,alokjani,amar,ynshenoy,abhishekl,mayankkapoor,soumit,anshup,varunarya,prashant,punituee,saju,ajayaa,shankc]
+rjil::system::accounts::sudo_users: [soren,bodepd,hkumar,jenkins,consul,pandeyop,jaspreet,vivek,ahmad,vaidy,himanshu,rohit,sanjayu,alokjani,amar,ynshenoy,abhishekl,mayankkapoor,soumit,anshup,varunarya,prashant,punituee,saju,ajayaa,shankc]
```
Comment (mayankkapoor) [inline hiera/data/env/at.yaml]: I think your usernames cannot be different. If you stick with shankc here, you will need to stick with shankc in common.yaml. Not completely sure about this. Best to ping harish/anshu about this.

File: hiera/data/secrets/common.yaml [hunk 2]
```diff
@@ -23,6 +23,7 @@ rjil::system::accounts::local_users:
...
   ajayaa: {realname: 'Ajaya Agrawal', sshkeys: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDs66WwKJJ/rUrwgPrACPIXqmiaIPTbU1tg+A8rYZW8WhDeTvqFYbHqfo/KAVtExWiV7wdKO7tltyR27Vp+43T1j3rO+u9Gwg0uzNpW418WVvgE01tCvAoFIxSsPdeXjpFj6KBvEdF8vTjmY4srLzx1H1oxMHYRKcOZbFlM/w4TnimT5nlLjWK63GV7QTebN/2TRfPO2pOQmI6wK8BO5aMY8GZ8/8UjLcKG2wcykjhpY1UxdrrrR5kVXUTrx1ZrMBEZuWHgARS5RFe15ZDNo12ng9vyLNHkdo5gvMXR6vLKghKH3XdkglMo03KjaTzwSFfHtk0ddyScjvDUd2LXZG03 aj@winterfell", password: '*'}
+  shashankchakelam: {realname: 'Shashank Chakelam', sshkeys: "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQC4LEHHA0qn7IAp+1E5bQ/lap3RlzalDLptXA8J22RcHMZr8+MFo4sxQnFTNIABRwzNqhDck3+o7zp2HZB/6VBbtlfFNVgVwQlYZrzbNyGe9160bzUXgV3BIAr9xfLUGcb/TB+E6E9/49HUHx18kwt5X3rwki6aG5hO/1fPtpI0EK++sXcSVYdmNnFm7kMQK54fLuL2GZPJD+y35QUMvvZyiL6VF573FPo5bPuohwtV3UxWvUTevQ4gEjyEnwRth7rxv3ZNWKVp4kk0xytrOQNF3qVxUMwmrjfCZuY63kwq9Rzzdvpmy9pTVVNGoMAxlv6fWtZMkIxVVpPfZOh86Igt shank@shankUbuntuLap", password: '*'}
```
Comment (mayankkapoor) [inline hiera/data/secrets/common.yaml]: See above comment.

File: hiera/data/env/at.yaml (hunk 1 above)
Comment (shashankchakelam) [inline hiera/data/env/at.yaml]: Changed the username to shashankchakelam in both cases

**Comment by mayankkapoor** [comment]
please test this

**Comment by bodepd** [comment]
why does this require 4 commits to change 2 lines of configuration?

**Comment by bodepd** [comment]
I'm going to close as well since it needs to be rebased.

**Pull request closed without merging**